2024-08-07 18:18:19,094 INFO    StreamThr :23334 [internal.py:wandb_internal():86] W&B internal server running at pid: 23334, started at: 2024-08-07 18:18:19.094272
2024-08-07 18:18:19,094 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: status
2024-08-07 18:18:19,095 INFO    WriterThread:23334 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/run-cerri7wl.wandb
2024-08-07 18:18:19,096 DEBUG   SenderThread:23334 [sender.py:send():382] send: header
2024-08-07 18:18:19,115 DEBUG   SenderThread:23334 [sender.py:send():382] send: run
2024-08-07 18:18:19,367 INFO    SenderThread:23334 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files
2024-08-07 18:18:19,367 INFO    SenderThread:23334 [sender.py:_start_run_threads():1136] run started: cerri7wl with start time 1723025899.094226
2024-08-07 18:18:19,379 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 18:18:19,379 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: check_version
2024-08-07 18:18:19,487 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 18:18:19,499 DEBUG   HandlerThread:23334 [system_info.py:__init__():32] System info init
2024-08-07 18:18:19,499 DEBUG   HandlerThread:23334 [system_info.py:__init__():47] System info init done
2024-08-07 18:18:19,499 INFO    HandlerThread:23334 [system_monitor.py:start():194] Starting system monitor
2024-08-07 18:18:19,500 INFO    SystemMonitor:23334 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 18:18:19,500 INFO    HandlerThread:23334 [system_monitor.py:probe():214] Collecting system info
2024-08-07 18:18:19,501 INFO    SystemMonitor:23334 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 18:18:19,503 INFO    SystemMonitor:23334 [interfaces.py:start():190] Started disk monitoring
2024-08-07 18:18:19,505 INFO    SystemMonitor:23334 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 18:18:19,506 INFO    SystemMonitor:23334 [interfaces.py:start():190] Started memory monitoring
2024-08-07 18:18:19,507 INFO    SystemMonitor:23334 [interfaces.py:start():190] Started network monitoring
2024-08-07 18:18:19,522 DEBUG   HandlerThread:23334 [system_info.py:probe():196] Probing system
2024-08-07 18:18:19,523 DEBUG   HandlerThread:23334 [system_info.py:_probe_git():181] Probing git
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_info.py:_probe_git():189] Probing git done
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_info.py:probe():244] Probing system done
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T10:18:19.522461', 'startedAt': '2024-08-07T10:18:19.092751', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/llama2-7b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 927.919607142857, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 1082.421, 'min': 800.0, 'max': 5500.0}, {'current': 1114.116, 'min': 800.0, 'max': 5500.0}, {'current': 1280.817, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1269.869, 'min': 800.0, 'max': 5500.0}, {'current': 1097.878, 'min': 800.0, 'max': 5500.0}, {'current': 1413.805, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1294.183, 'min': 800.0, 'max': 5600.0}, {'current': 1096.493, 'min': 800.0, 'max': 5600.0}, {'current': 1379.317, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1119.065, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1200.569, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1142.66, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.6722068786621}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-07 18:18:19,528 INFO    HandlerThread:23334 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 18:18:19,528 INFO    HandlerThread:23334 [system_monitor.py:probe():227] Publishing system info
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 18:18:19,528 DEBUG   HandlerThread:23334 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 18:18:20,368 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/conda-environment.yaml
2024-08-07 18:18:20,368 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/requirements.txt
2024-08-07 18:18:21,205 DEBUG   HandlerThread:23334 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 18:18:21,205 DEBUG   HandlerThread:23334 [system_info.py:_save_code():90] Saving code
2024-08-07 18:18:21,206 DEBUG   HandlerThread:23334 [system_info.py:_save_code():111] Saving code done
2024-08-07 18:18:21,206 DEBUG   HandlerThread:23334 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 18:18:21,245 DEBUG   HandlerThread:23334 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 18:18:21,246 INFO    HandlerThread:23334 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 18:18:21,248 DEBUG   SenderThread:23334 [sender.py:send():382] send: files
2024-08-07 18:18:21,248 INFO    SenderThread:23334 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 18:18:21,249 INFO    SenderThread:23334 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 18:18:21,250 INFO    SenderThread:23334 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 18:18:21,256 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 18:18:21,257 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 18:18:21,259 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: stop_status
2024-08-07 18:18:21,367 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/conda-environment.yaml
2024-08-07 18:18:21,367 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/code/llama/gmm.py
2024-08-07 18:18:21,367 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/wandb-metadata.json
2024-08-07 18:18:21,367 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/diff.patch
2024-08-07 18:18:21,368 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/code/llama
2024-08-07 18:18:21,368 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/code
2024-08-07 18:18:21,508 DEBUG   SenderThread:23334 [sender.py:send():382] send: telemetry
2024-08-07 18:18:21,508 DEBUG   SenderThread:23334 [sender.py:send():382] send: config
2024-08-07 18:18:21,510 DEBUG   SenderThread:23334 [sender.py:send():382] send: metric
2024-08-07 18:18:21,510 DEBUG   SenderThread:23334 [sender.py:send():382] send: telemetry
2024-08-07 18:18:21,510 DEBUG   SenderThread:23334 [sender.py:send():382] send: metric
2024-08-07 18:18:21,510 WARNING SenderThread:23334 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 18:18:21,510 DEBUG   SenderThread:23334 [sender.py:send():382] send: telemetry
2024-08-07 18:18:21,511 DEBUG   SenderThread:23334 [sender.py:send():382] send: telemetry
2024-08-07 18:18:21,511 DEBUG   SenderThread:23334 [sender.py:send():382] send: config
2024-08-07 18:18:21,514 DEBUG   SenderThread:23334 [sender.py:send():382] send: exit
2024-08-07 18:18:21,514 INFO    SenderThread:23334 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 18:18:21,515 INFO    SenderThread:23334 [sender.py:send_exit():591] handling runtime: 1
2024-08-07 18:18:21,515 INFO    SenderThread:23334 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:18:21,517 INFO    SenderThread:23334 [sender.py:send_exit():597] send defer
2024-08-07 18:18:21,518 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,518 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 18:18:21,520 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,520 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 18:18:21,520 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 1
2024-08-07 18:18:21,520 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,521 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 18:18:21,521 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,521 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 18:18:21,521 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 2
2024-08-07 18:18:21,521 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,521 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 18:18:21,521 INFO    HandlerThread:23334 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 18:18:21,521 DEBUG   SystemMonitor:23334 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 18:18:21,522 INFO    HandlerThread:23334 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 18:18:21,522 DEBUG   SystemMonitor:23334 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 18:18:21,522 DEBUG   SystemMonitor:23334 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 18:18:21,525 INFO    HandlerThread:23334 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 18:18:21,546 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3
2024-08-07 18:18:21,546 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5215', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:21,546 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AEDAFF9C48D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [interfaces.py:finish():202] Joined network monitor
2024-08-07 18:18:21,559 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 3
2024-08-07 18:18:21,559 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,559 DEBUG   SenderThread:23334 [sender.py:send():382] send: stats
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 18:18:21,559 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 4
2024-08-07 18:18:21,559 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 18:18:21,559 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 18:18:21,559 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 5
2024-08-07 18:18:21,559 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,559 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 18:18:21,559 DEBUG   SenderThread:23334 [sender.py:send():382] send: summary
2024-08-07 18:18:21,560 INFO    SenderThread:23334 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:18:21,560 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,560 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 18:18:21,560 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 6
2024-08-07 18:18:21,560 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,560 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 18:18:21,560 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,560 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 18:18:21,561 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 18:18:21,595 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e
2024-08-07 18:18:21,595 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:21,595 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AEDB062FC45</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:21,694 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f
2024-08-07 18:18:21,695 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287190', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:21,695 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AEDABD2E73B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:21,730 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 7
2024-08-07 18:18:21,730 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:21,730 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 18:18:21,730 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:21,730 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 18:18:22,368 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/config.yaml
2024-08-07 18:18:22,369 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/wandb-summary.json
2024-08-07 18:18:22,369 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/output.log
2024-08-07 18:18:22,454 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:18:22,630 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3
2024-08-07 18:18:22,631 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5215', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:22,631 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AEDF05ED163</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:22,837 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e
2024-08-07 18:18:22,837 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:22,838 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AEDFCAFB02C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:22,838 INFO    wandb-upload_1:23334 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:18:23,013 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f
2024-08-07 18:18:23,013 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287190', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:23,013 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AEE04E2B36B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:23,525 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 8
2024-08-07 18:18:23,525 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 18:18:23,526 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:23,526 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 18:18:23,526 DEBUG   SenderThread:23334 [sender.py:send_request():409] send_request: defer
2024-08-07 18:18:23,526 INFO    SenderThread:23334 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 18:18:23,526 INFO    SenderThread:23334 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 18:18:23,526 INFO    SenderThread:23334 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 18:18:23,542 INFO    SenderThread:23334 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 18:18:23,656 INFO    SenderThread:23334 [sender.py:transition_state():617] send defer: 9
2024-08-07 18:18:23,656 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:18:23,656 INFO    HandlerThread:23334 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 18:18:23,657 DEBUG   SenderThread:23334 [sender.py:send():382] send: artifact
2024-08-07 18:18:24,239 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401
2024-08-07 18:18:24,239 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:24,240 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AEE5078DA69</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:24,369 INFO    Thread-12 :23334 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181819-cerri7wl/files/output.log
2024-08-07 18:18:24,383 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838
2024-08-07 18:18:24,384 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:24,384 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AEE540A2A2F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:24,834 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3
2024-08-07 18:18:24,834 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5215', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:24,834 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AEE73C1E82D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:24,936 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e
2024-08-07 18:18:24,936 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:24,936 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AEE79C7BA2F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:25,318 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401
2024-08-07 18:18:25,318 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:25,318 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AEE90B375EA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:25,579 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f
2024-08-07 18:18:25,579 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287190', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:25,579 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AEE9E4FF09C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:25,645 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838
2024-08-07 18:18:25,645 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:25,646 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AEE9F6160D5</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:27,456 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:18:27,796 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401
2024-08-07 18:18:27,796 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:27,796 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AEF24557A30</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:27,796 INFO    wandb-upload_3:23334 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:18:27,850 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838
2024-08-07 18:18:27,850 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:27,850 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AEF25D38E5B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:27,850 INFO    wandb-upload_4:23334 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:18:29,107 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e
2024-08-07 18:18:29,107 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:29,107 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AEF726A41CF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:29,444 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3
2024-08-07 18:18:29,445 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5215', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:29,445 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AEF868FB7B9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:30,147 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f
2024-08-07 18:18:30,147 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287190', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:30,147 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AEFAEAC8D55</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:32,360 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838
2024-08-07 18:18:32,360 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:32,360 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF032CD011E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:32,458 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:18:32,681 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401
2024-08-07 18:18:32,681 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:32,681 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF0477DF9F1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:37,213 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4524526397938a35152f286085e7e5279410ac33b3bdacef9e1b5e593e57ec6e
2024-08-07 18:18:37,213 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:37,214 ERROR   wandb-upload_1:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AF15594BD78</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:37,460 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:18:38,273 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=901f52002004c6a23db6f62cb0784d692c20bd166415a0797539e479e73de5d3
2024-08-07 18:18:38,273 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5215', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:38,273 ERROR   wandb-upload_0:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AF194B3381A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:40,150 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/cerri7wl/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101821Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=256292a9dde7024f339d7d9d7db7ab2951a865c4f8e6c38ecc5daf5500606a7f
2024-08-07 18:18:40,150 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287190', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:40,150 ERROR   wandb-upload_2:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/cerri7wl/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/cerri7wl/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AF202F14358</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:42,154 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=22ceb516ca813f4df16218fd367201a2b03bb9290cd65ab99cb991cb4f045838
2024-08-07 18:18:42,155 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:42,155 ERROR   wandb-upload_4:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF2767B0888</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:42,385 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101824Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b418af3dc9d51b3c9c670e6b0acd949a88db45e21968f938c2152c53bc3ef401
2024-08-07 18:18:42,385 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:18:42,385 ERROR   wandb-upload_3:23334 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/507/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF289EAC598</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:18:42,462 DEBUG   HandlerThread:23334 [handler.py:handle_request():146] handle_request: keepalive
